### PR TITLE
[Beat] Add Reset Balance Account on Face Payment

### PR DESCRIPTION
**Pull request by lazuardyk**
## Type of changes

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?
There is no API for reset balance face payment account

## What is the new behavior?
Added endpoint for reset balance face payment account to 1000000 (default)

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
Screenshot:
Screenshot_2005

### CODE REVIEW

File: backend/controllers/face_payment.go [hunk 1]
```diff
@@ -203,6 +203,48 @@ func (ctrl *Controller) CreateFacePaymentAccount(ctx *gin.Context) {
...
+		ctx.JSON(http.StatusNotFound, gin.H{
+			"ok":      false,
+			"message": "This session id does not have an active face payment account",
+		})
+		return
+	}
+
+	var accountWallet models.FacePaymentWallet
+	ctrl.Model.GetAccountWallet(&accountWallet, account.ID)
+	accountWallet.Balance = 1000000
```
Comment (sonadztux) [inline backend/controllers/face_payment.go:239]: Ini dijadiin constant aja ga, sih?

Comment (lazuardyk) [reply backend/controllers/face_payment.go:239]: boleh

Comment (lazuardyk) [reply backend/controllers/face_payment.go]: done https://github.com/nodefluxio/weber/pull/133/commits/dc60fffe99fa5db1474469ece356e73038421ee1

**Review by sonadztux — APPROVED** [review]
lgtm keep it up!

**Review by bagasteguhimani — APPROVED** [review]
Work!

**Pull request merged**